docs/useful-libraries/react/README.md

```diff
@@ -2,25 +2,25 @@
 
 ## General
 
-* [Reach UI](https://reacttraining.com/reach-ui/)
+* [Reach UI](https://reacttraining.com/reach-ui/) - Accessible foundation for React apps
 
 ## Hooks
 
-* [react-use-measure](https://github.com/react-spring/react-use-measure)
-* [react-intersection-observer](https://github.com/thebuilder/react-intersection-observer)
-* [react-use](https://github.com/streamich/react-use)
-* [react-adaptive-hooks](https://github.com/GoogleChromeLabs/react-adaptive-hooks)
-* [use-persisted-state](https://github.com/donavon/use-persisted-state)
-* [use-query-string](https://github.com/trevorblades/use-query-string)
-* [promise-hook](https://github.com/aiven715/promise-hook)
-* [react-hotkeys-hook](https://github.com/JohannesKlauss/react-hotkeys-hook)
+* [react-use-measure](https://github.com/react-spring/react-use-measure) - Reactive boundry measurments
+* [react-intersection-observer](https://github.com/thebuilder/react-intersection-observer) - Check if element is in the viewport
+* [react-use](https://github.com/streamich/react-use) - Large collection of essential hooks
+* [react-adaptive-hooks](https://github.com/GoogleChromeLabs/react-adaptive-hooks) - Hooks and utilities for adaptive loading
+* [use-persisted-state](https://github.com/donavon/use-persisted-state) - Multi-instance, multi-tab/browser shared and persistent state
+* [use-query-string](https://github.com/trevorblades/use-query-string) - Serialize state into the URL query string
+* [promise-hook](https://github.com/aiven715/promise-hook) - Simple data fetching
+* [react-hotkeys-hook](https://github.com/JohannesKlauss/react-hotkeys-hook) - Simple keyboard shortcut support
 
 ## Components
 
-* [downshift](https://github.com/downshift-js/downshift)
-* [react-focus-lock](https://github.com/theKashey/react-focus-lock)
-* [formik](https://github.com/jaredpalmer/formik)
-* [react-beautiful-dnd](https://github.com/atlassian/react-beautiful-dnd)
+* [downshift](https://github.com/downshift-js/downshift) - Autocomplete/dropdown/select/combobox components
+* [react-focus-lock](https://github.com/theKashey/react-focus-lock) - Browser friendly focus lock
```

Comment:
What is a focus lock? When would I use it? Also I think "browser friendly" is redundant as everything here is for use in a web browser